llvm_config_linux.go

```diff
@@ -9,4 +9,6 @@ package llvm
 // #cgo LDFLAGS: -L/usr/lib/llvm-9/lib  -lLLVM-9
 import "C"
 
-type (run_build_sh int)
+type (
+	run_build_sh int
+)
```

Comment:
The intention is that this file can always be regenerated with a `make config` so the Makefile should also have a similar change to correctly format the source code.

Reply:
The easiest way to do this is probably to remove the parentheses in this line:

https://github.com/tinygo-org/go-llvm/blob/1ff21df335666b54d9b3c601c11800285e00939d/Makefile#L55

Reply:
OK, I've resolved that in #12.